win-linux/package/windows/common.iss

```diff
@@ -1,54 +1,136 @@
+  #define sPlatform                 "win_64"
+#elif str(_ARCH) == "32"
+  #define sWinArch                  "x86"
+  #define sPlatform                 "win_32"
+#endif
+
+#ifndef sAppIconName
+  #define sAppIconName              "ONLYOFFICE Editors"
+#endif
+#ifndef MAIN_EXE
+  #define MAIN_EXE                  "DesktopEditors.exe"
+#endif
+#define APP_PATH                    str(sIntCompanyName + "\" + sIntProductName)
+#define APP_REG_PATH                str("Software\" + APP_PATH)
+#ifdef _MEDIAVIEWER
+  #define MEDIAVIEWER_APP_PATH      str(sIntCompanyName + "\MediaViewer")
+  #define IMAGEVIEWER_APP_REG_PATH  str("Software\" + sIntCompanyName + "\ImageViewer")
+  #define VIDEOPLAYER_APP_REG_PATH  str("Software\" + sIntCompanyName + "\VideoPlayer")
+#endif
+#ifndef APP_USER_MODEL_ID
+  #define APP_USER_MODEL_ID         "ASC.Documents.5"
+#endif
+#ifndef NAME_EXE_IN
+  #define NAME_EXE_IN               str(sIntProductName + "_" + sWinArch + ".exe")
+#endif
+#ifndef NAME_EXE_OUT
+  #define NAME_EXE_OUT              "editors.exe"
+#endif
+#ifndef LIC_FILE
+  #define LIC_FILE                  "agpl-3.0"
+#endif
+#ifndef APP_MUTEX_NAME
+  #define APP_MUTEX_NAME            "TEAMLAB"
+#endif
+
+#define APPWND_CLASS_NAME           "DocEditorsWindowClass"
+#define VISEFFECTS_MANIFEST_NAME    ChangeFileExt(MAIN_EXE, "VisualElementsManifest.xml")
 
 #ifndef SCRIPT_CUSTOM_FILES
-#  define sAppVersion         GetFileVersion(AddBackslash(SourcePath) + '..\..\Build\Release\' + NAME_EXE_IN)
+  #define sAppVersion               GetFileVersion(AddBackslash(SourcePath) + "..\..\Build\Release\" + NAME_EXE_IN)
 #else
-#  ifdef _WIN_XP
-#    define xp_suffix  '_xp'
-#  else
-#    define xp_suffix
-#  endif
-#  define DEPLOY_PATH '..\..\..\..\build_tools\out\' + os_arch + xp_suffix + '\ONLYOFFICE'
-#  define sAppVersion         GetFileVersion(AddBackslash(DEPLOY_PATH) + 'DesktopEditors\' + NAME_EXE_OUT)
+  #ifndef _WIN_XP
+    #define DEPLOY_PATH             str("..\..\..\..\build_tools\out\" + sPlatform + "\" + sIntCompanyName)
+  #else
+    #define DEPLOY_PATH             str("..\..\..\..\build_tools\out\" + sPlatform + "_xp\" + sIntCompanyName)
+  #endif
+  #define sAppVersion               GetFileVersion(AddBackslash(DEPLOY_PATH) + "DesktopEditors\" + NAME_EXE_OUT)
 #endif
-
-#define sAppVerShort        Copy(sAppVersion, 0, 3)
+#define sAppVerShort                Copy(sAppVersion, 0, 3)
+;#ifndef sAppVerShort
+;  #define sAppVerShort              "0.0.0"
+;#endif
+;#ifndef sAppBuildNumber
+;  #define sAppBuildNumber           "0"
+;#endif
+;#ifndef sAppVersion
+;  #define sAppVersion               str(sAppVerShort + "." + sAppBuildNumber)
+;#else
+;  
+;#endif
 
 #include "utils.iss"
 #include "associate_page.iss"
 
 #define UNINSTALL_USE_CLEAR_PAGE
 #ifdef UNINSTALL_USE_CLEAR_PAGE
-# include "uninstall_page.iss"
+  #include "uninstall_page.iss"
 #endif
 
+#ifdef _MEDIAVIEWER
+  #include "..\..\..\..\core-ext\multimedia\packages\exe\base.iss"
```

Comment:
Use [this](https://github.com/ASC-OFFICE/r7/blob/develop/multimedia/packages/exe/base.iss) link insttead of core-ext